GETTING-STARTED.md

```diff
@@ -114,7 +114,15 @@ pyscript.write('pi', f'π is approximately {pi:.3f}')
 In addition to the [Python Standard Library](https://docs.python.org/3/library/) and
 the `pyscript` module, many 3rd-party OSS packages will work out-of-the-box with PyScript.
 In order to use them you will need to delcare the dependencies using the `<py-env>` in the
-HTML head.
+HTML head. You can also link to `.whl` files directly on disk like in our [toga example](https://github.com/pyscript/pyscript/blob/main/pyscriptjs/examples/toga/freedom.html)
+
+```
+<py-env>
+- './static/wheels/travertino-0.1.3-py3-none-any.whl'
+</py-env>
+```
+
+If your `.whl` is not a pure Python wheel then open a PR or issue with [pyodide](https://github.com/pyodide/pyodide) to get it added here https://github.com/pyodide/pyodide/tree/main/packages
```

Comment:
I'm not sure this is the expectations from the Pyodide team. @rth @hoodmane, what do you think?

Reply:
Yes, that's OK. We are trying to make this easier but for now, that's indeed what one has to do.

Reply:
Opening a PR or issue on the Pyodide issue tracker is good advice. It might be worth commenting that if there is popular demand from 3+ different users the Pyodide team is likely to work on porting the package, if we haven't received that much interest yet you will have to write your own PR. Of course things will be faster if you work on a PR, we like to collaborate with newcomers on ports (we may provide advice, help you if you get stuck, and or step in and work on it ourselves if it turns out it's hard in an interesting way).

We are working towards out of tree builds but unfortunately they aren't ready yet.